test/rpc/RPC.test.ts

```diff
@@ -164,10 +289,8 @@ describe('RPC', () => {
   //   const eraSummary = await client.getEraInfoBySwitchBlockHeight(10);
```

Comment:
can we uncomment this code and make it working? I mean to check if we can find the deploys that was done before  in the erc20 section?